src/third-party.md

```diff
@@ -36,6 +36,16 @@ Snap it into your existing workflows with the click of a button, automate away t
 * [The Last Pickle Cassandra stress tool, tlp-stress](https://github.com/thelastpickle/tlp-stress): A workload-centric stress tool for Apache Cassandra. Designed for simplicity, no math degree required. (DataStax)
 * [NoSQLBench](https://github.com/nosqlbench/nosqlbench): Pluggable benchmarking suite for Cassandra and other distributed systems
 * [Stargate](https://stargate.io): Open source data gateway providing Document, REST, and GraphQL APIs to Apache Cassandra.
+* [Instaclustr Esop](https://github.com/instaclustr/esop): Swiss knife for backup and restore of your node to GCP, Azure, S3, Ceph etc. Supports backup and restoration of commit logs too. Esop is embedded in [Instaclustr Icarus](https://github.com/instaclustr/icarus) sidecar so you may backup and restore your cluster remotely and on-the-fly with any disruption.
```

Comment:
Esop is embedded in [Instaclustr Icarus](https://github.com/instaclustr/icarus) sidecar for remote cluster on-the-fly backup and restore with any disruption.

And is that supposed to be "without any disruption"?